liwords-ui/src/gameroom/table.tsx

```diff
@@ -873,13 +874,21 @@ export const Table = React.memo((props: Props) => {
         : singularCount(n, 'Player', 'Players'),
     [isObserver]
   );
+  const boardTheme =
+    'board--' + tournamentContext.metadata.getBoardStyle() || '';
+  const tileTheme = 'tile--' + tournamentContext.metadata.getTileStyle() || '';
```

Comment:
I think these need to be in parentheses: 

```('tile--' + tournamentContext.metadata.getTileStyle()) || '';``` otherwise the result looks like `tile--` if getTileStyle is empty. same for boardTheme above it.